vendor/bat-native-ads/src/bat/ads/internal/client_state.cc

```diff
@@ -95,6 +95,12 @@ Result ClientState::FromJson(
 
   if (client.HasMember("adsShownHistory")) {
     for (const auto& ad_shown : client["adsShownHistory"].GetArray()) {
+      // adsShownHistory used to be an array of timestamps, so if
+      // that's what we have here don't import them and we'll just
+      // start fresh.
+      if (ad_shown.GetUint64()) {
```

Comment:
Should use `IsUint64()`

Reply:
Oh boy, that's what I meant to type but my fingers didn't listen. Fixing!